paper.tex

```diff
@@ -2470,47 +2470,54 @@ \section{Cell lineages and ARGs}
-% It may be worth putting in something here about events being a slippery concept
-% (we have already mentioned this at the end of Section 3
 Here, an event is not represented by a specific
-genome. Rather, genomes can be associated with (or ``tag'') events above or below them (and as argued in
-Section~\ref{sec-precision}, the gARG representation allows these associated genomes to be arbitrarily distant
-in time or number of cell
-divisions from the event itself). For example, tagging the two genomes above a recombination event leads to the
-2 node representation seen in Figs.~\ref{fig-simplification} and \ref{hudson_vs_bigARG}, whereas tagging the genome
-below a recombination event leads to the more conventional graphs in Figs.~\ref{fig-ancestry-resolution} and \ref{fig-inferred-args}A,B.
-
-Thick black lines trace an ARG that describes the ancestry of the lower arm of the
-four sampled chromosomes, and these lines pass through gARG nodes (green chromosomal regions).
-A number of these nodes have only one parent and one child in the ARG; these would often be omitted
-(taken as implicit) in a minimally simplified gARG.
-
-In particular, this schematic illustrates a case where three ARG lineages coalesce within
-a single individual ($D_{10}$), corresponding to what is normally represented by a polytomy
-(e.g. Fig.~\ref{fig-simplification} node \noderef{k}). The process that generates
-these three lineages actually consists of two successive bifurcations, reflecting the fact that
-the only known method of reproducing DNA is by (semi-conservative) duplication. Under this schematic,
-polytomies therefore represent a gARG-enabled merging of multiple coalescent events into an older
-node, just as ARG nodes with more than 2 parents represent a merging of multiple recombination
-events into a younger node.
-
-Note that mutations can occur along any lineage in Fig.~\ref{fig-cell-lines}. This means, for example,
-that a mutation in the first cell division of $D_{10}$ could be shared between
-the two gametes produced by the cells in the left half of $D_{10}$ but not shared by
-the right hand gamete. With a high enough mutation rate (or equivalently, a long enough genome), each
+genome. Rather, genomes can be associated with, or ``tag'', events above
+(ancestral to) or below (descended from) them.
+For example, tagging the two genomes above a recombination event leads to the
+two node representation seen in
+Figs.~\ref{fig-simplification} and \ref{hudson_vs_bigARG},
+whereas tagging the genome
+below a recombination event leads to the more
+conventional graphs in Figs.~\ref{fig-ancestry-resolution} and \ref{fig-inferred-args}A,B.
+
+%% JK:  What does this add that is not in caption?
+% Thick black lines trace an ARG that describes the ancestry of the lower arm of
+% the four sampled chromosomes, and these lines pass through gARG nodes (green
+% chromosomal regions). A number of these nodes have only one parent and one
```

Comment:
Yes, you are right - I made a slight clarification to the caption just to make sure. I think the pass-through thing can be omitted.